### PR TITLE
Mod/11028 dynamic dsm height

### PR DESCRIPTION
**Pull request by jagu65**
**High level description:**
Fixed the dynamic height of the DS&M accordion.

**Technical details:**
Added a useState and a useEffect to record the height of the content element and update the height of the DS&M element.

**JIRA Ticket:**
[DEV-11028](https://federal-spending-transparency.atlassian.net/browse/DEV-11028)

**Mockup:**
n/a

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [ ] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [ ] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [ ] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

File: src/_scss/pages/search/resultsView/_resultsView.scss [hunk 1]
```diff
@@ -227,16 +227,22 @@
 
         h4 {
           margin: rem(24) 0 rem(8) 0;
-          font-size: 16px;
           font-weight: $font-semibold;
           color: $gray-90;
-          line-height: rem(24);
+          font-size: 14px;
```
Comment (anjenkin) [inline src/_scss/pages/search/resultsView/_resultsView.scss:232]: Please use rem

File: src/_scss/pages/search/resultsView/_resultsView.scss [hunk 2]
```diff
@@ -227,16 +227,22 @@
...
+          line-height: 1.5;
         }
 
         p {
+          font-size: 14px;
```
Comment (anjenkin) [inline src/_scss/pages/search/resultsView/_resultsView.scss:237]: Same as above

File: src/_scss/pages/search/resultsView/_resultsView.scss (hunk 1 above)
Comment (jagu65) [reply src/_scss/pages/search/resultsView/_resultsView.scss]: Merged in QAT which uses rem

File: src/_scss/pages/search/resultsView/_resultsView.scss (hunk 2 above)
Comment (jagu65) [reply src/_scss/pages/search/resultsView/_resultsView.scss]: Merged in QAT which uses rem

**Review by anjenkin — APPROVED** [review]

**Pull request merged**